pyof/foundation/basic_types.py

```diff
@@ -430,8 +435,10 @@ def pack(self, value=None):
         if value is None:
```

Comment:
I don't think so, because, in this specific case:
``None == 0 == b'' == ''`` (`==` -> behaves like)